packages/ckeditor5-editor-multi-root/src/multirooteditoruiview.ts

```diff
@@ -65,6 +67,7 @@ export default class MultiRootEditorUIView extends EditorUIView {
 		options: {
 			editableElements?: Record<string, HTMLElement>;
 			shouldToolbarGroupWhenFull?: boolean;
+			label?: string | Record<string, string>;
```

Comment:
Shouldn't it be just `label?: Record<string, string>;`? Or one can pass `string` and then all roots will have same label (and if so, is it correct behavior from accessibility perspective?).

Reply:
Yes, you can pass string here and all roots will get the same name. Is it accessible? It's hard to tell. `aria-label` does not have to be unique and can be repeated on a website. 

Multiple roots can be used creatively, for instance, only one visible/reachable at a time so this approach could be valid. This is also useful if many new roots are created on the fly (e.g. editing a book, a chapter per root, but only one root is visible at a time). I even considered making it `label?: string | Record<string, string> | ( ( name: string ) => string )` for this purpose ("Rich text editor. Editing area: Chapter 11, foo bar baz") but I figured this would be a scope creep and an easy improvement in the future.

Anyway, I mirrored the [placeholder configuration](https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editorconfig-EditorConfig.html#member-placeholder) (also `string | Record<string, string>`) and implementation. 

https://github.com/ckeditor/ckeditor5/blob/54efa9938f46463d11f450b009b63140fe9bf946/packages/ckeditor5-editor-multi-root/src/multirooteditorui.ts#L203-L205

If you think we should escalate it and discuss it with more team members, I'm OK with that 🙂

Reply:
Thanks for detailed explanation 👍 

> `aria-label` does not have to be unique and can be repeated on a website.

I think the above solves it since this was my main concern here I guess. And the use case you mentioned is valid so it may be desired behavior in such cases 👍 

> I even considered making it `label?: string | Record<string, string> | ( ( name: string ) => string ) for this purpose ("Rich text editor. Editing area: Chapter 11, foo bar baz")`

Hah, I had similar idea but hold myself off from bringing it up too 👍 😄